### ISSUE TITLE
Doesn't Track Time When Window Tiling Feature Is Enabled

### ISSUE THREAD

**Issue by ankushdas9** [ActivityWatch/activitywatch#432]
- [x] I am on the [latest](https://github.com/ActivityWatch/activitywatch/releases/latest) ActivityWatch version.
 - [x] I have searched the issues of this repo and believe that this is not a duplicate.

- **OS version and name**: Pop!_OS 20.04
- **ActivityWatch version**: Version: v0.9.2

## Describe the bug
So, when the window tiling feature is enabled on Pop!_OS 20.04 (it's a default feature on Pop!_OS 20.04) ActivityWatch fails to record anything at all. 

Here's a screenshot of my activity for reference: https://imgur.com/a/5tb7DKI

For some reason it was able to record a few seconds of my time on Rocket.chat app - but other than that there's nothing else.

## To Reproduce
1. Enable window tiling feature.
2. Emulate some activity
3. Check ActivityWatch activity.

## Expected behavior
I'm not sure if this is technically possible for ActivityWatch — but if it can somehow gauge the time spent even with Window tiling feature is enabled that would be great!

## Documentation
Not sure what exact log file should I upload here - let me know what you need.

## Additional context
The window tiling feature on Pop!_OS also highlights the active window — so if it can detect the active window, it might get the data needed.

**Comment by ErikBjare** [comment]
Please attach the aw-watcher-window logs

**Comment by johan-bjareholt** [comment]
> more details about what this "window tiling" feature does differently that might affect it (a screenshot would be nice).

I assume he refers to this

https://github.com/pop-os/shell

Since it's just an extension to gnome-shell it sounds very strange that it would break aw-watcher-window, if that's true they must be doing something very strange under the hood.

@nicolae-stroncea I was browsing pop-shell issues and found one issue where I recognized your name since you have contributed to activitywatch before. Does activitywatch work for you with pop-shell?

**Comment by ankushdas9** [comment]
I noticed that after I disabled the auto-tiling feature and tried to access to ActivityWatch dashboard again, I found the recorded activity.

Not sure if it failed to refresh back then or if it was a conflict with the auto-tiling feature.

So, I have attached some recent logs here. You can let me know what exactly happened in my case.

[aw-watcher-window_2020-05-18T14-09-41.log](https://github.com/ActivityWatch/activitywatch/files/4643114/aw-watcher-window_2020-05-18T14-09-41.log)
[aw-watcher-window_2020-05-16T05-33-49.log](https://github.com/ActivityWatch/activitywatch/files/4643116/aw-watcher-window_2020-05-16T05-33-49.log)

> I assume he refers to this
> 
> https://github.com/pop-os/shell

And, yes, that would give you all the information about the auto-tile mode on Pop!_OS 20.04. If you need anything else, let me know!

Thanks!

**Comment by ErikBjare** [comment]
>I noticed that after I disabled the auto-tiling feature and tried to access ActivityWatch dashboard again, I found the recorded activity.
>
>Not sure if it failed to refresh back then or if it was a conflict with the auto-tiling feature.

So you now have activity for the period you were missing before? Or just when the tiling feature was off?

**Comment by ankushdas9** [comment]
> > I noticed that after I disabled the auto-tiling feature and tried to access ActivityWatch dashboard again, I found the recorded activity.
> > Not sure if it failed to refresh back then or if it was a conflict with the auto-tiling feature.
> 
> So you now have activity for the period you were missing before? Or just when the tiling feature was off?

Yes, now I can see the activity for the period I was missing before. 

However, the data refreshed automatically only after I disabled the tiling feature (I didn't try any manual refresh).

So, this could be a coincidence or the tiling feature could have been the culprit. I'll investigate this further with my use and report back the next day.

I'll close this issue if I find that it was a temporary cache-related (somehow didn't refresh) issue.

**Comment by nicolae-stroncea** [comment]
@johan-bjareholt  I've used pop shell for past 2 months (and ActivityWatch for a lot longer than that). I looked through my logs for that time period and everything looks normal.

**Comment by ErikBjare** [comment]
There's most likely no bug here, just an outdated Activity view. Closing.

**Comment by ankushdas9** [comment]
> There's most likely no bug here, just an outdated Activity view. Closing.
> 
> @ankushdas9 We welcome all bug reports, but please make sure you can actually replicate first, saves us time :)

Sure, my bad. Looks like after all the Activity didn't automatically refresh after 2 hours of my usage for some reason and wasn't related to window tiling feature at all.

I just thought I found something weird because after taking a closer look at the data I got after refreshing, I find some of my activity labelled as AFK — which wasn't the case.

Also, it didn't have any data for the previous day.

But, that's something different and I'll see if I can verify my observations before trying to raise the issue.

Keep up with all the good work! Thanks for your time! :)
